### DISCUSSION TITLE
使用anyviewer，手机无法远程控制电脑

### DISCUSSION THREAD

**Discussion by karteous** [kkkgo/PaoPaoDNS#143] [问答和帮助]
如题：使用了paopaodns+paopaogateway，并已在“force_recurse_list.txt”中加入：
domain:aomeisoftware.com
domain:anyviewer.com
domian:aomeikeji.com
还是无法实现远程控制

**Comment by kkkgo** [comment]
你的场景应该使用的是force_dnscrypt_list。参考https://github.com/kkkgo/PaoPaoDNS/discussions/122

**Reply by kkkgo** [reply]
另外不知道你的域名名单是来源于哪里，如果你是因为配合ppgw使用的导致流量跑到代理的话，你可以在网络流量记录功能看到被代理的域名记录。

**Reply by karteous** [reply]
域名名单是来源于网络流量记录，我也在force_dnscrypt_list里面加入试过，还是会无法连接远程电脑

**Reply by kkkgo** [reply]
检查下加入后是否生效（如编辑器或者权限原因），nslookup相关域名是否有被解析为fakeip。

**Reply by karteous** [reply]
加入后，网络流量记录不出现域名名单了，anyviewer显示的ip也是走了国内，但还是无法连接

**Reply by kkkgo** [reply]
修改第三方DNS（不使用ppdns）是否能正常使用？如果仍然不能正常使用，可能是anyviewer的问题或者网络问题。

**Reply by karteous** [reply]
不使用ppdns，应该是正常，因为我手机（使用ppdns）连接其他电脑（不使用ppdns）是可以正常使用的

**Reply by kkkgo** [reply]
手机（使用ppdns）连接其他电脑（不使用ppdns）是可以正常使用的：
你的意思是使用ppdns的电脑无法控制？那该电脑也不使用ppdns才能排除掉是dns的问题。
